### PR TITLE
{bio}[dummy/dummy] PLINK v2.00-alpha2-x86_64

### PR DESCRIPTION
**Pull request by murfreesboro**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by murfreesboro** [comment]
Test report by @murfreesboro
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
dev806.vampire - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.5
See https://gist.github.com/f4f2a7198d64887eff3e19fe8fc098c2 for a full test report.

**Comment by murfreesboro** [comment]
Test report by @murfreesboro
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
dev806.vampire - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.5
See https://gist.github.com/c0fc1e58e49ae310f4a81844c897f583 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nic92 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/ee11307d32d823308fa1b0841c3be6f7 for a full test report.

File: easybuild/easyconfigs/p/PLINK/PLINK-2.00-alpha2-x86_64_avx2.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+name = 'PLINK'
+version = '2.00-alpha2-x86_64_avx2'
+
+homepage = 'https://www.cog-genomics.org/plink/2.0/'
+description = 'Whole-genome association analysis toolset'
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://s3.amazonaws.com/plink2-assets/']
```
Comment (smoors) [inline easybuild/easyconfigs/p/PLINK/PLINK-2.00-alpha2-x86_64_avx2.eb]: @murfreesboro please use https here?

**Comment by murfreesboro** [comment]
Test report by @murfreesboro
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
dev806.vampire - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.5
See https://gist.github.com/5baffa4727e80298b71f8c37163ce41d for a full test report.

File: easybuild/easyconfigs/p/PLINK/PLINK-2.00-alpha2-x86_64.eb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+name = 'PLINK'
+version = '2.00-alpha2-x86_64'
+
+homepage = 'https://www.cog-genomics.org/plink/2.0/'
+description = 'Whole-genome association analysis toolset'
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://s3.amazonaws.com/plink2-assets/']
```
[the remark this answers is not in the document]
Comment (murfreesboro) [reply easybuild/easyconfigs/p/PLINK/PLINK-2.00-alpha2-x86_64.eb]: https is used.

File: easybuild/easyconfigs/p/PLINK/PLINK-2.00-alpha2-x86_64_avx2.eb (hunk 1 above)
Comment (murfreesboro) [reply easybuild/easyconfigs/p/PLINK/PLINK-2.00-alpha2-x86_64_avx2.eb]: https is used.

**Comment by smoors** [comment]
Going in, thanks @murfreesboro!

**Pull request merged**